src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenForManagedIdentityParameters.cs

```diff
@@ -16,6 +16,10 @@ internal class AcquireTokenForManagedIdentityParameters : IAcquireTokenParameter
 
         public string Resource { get; set; }
 
+        public string Claims { get; set; }
+
+        public string BadTokenHash { get; set; }
```

Comment:
Let's maybe use the term "RevokedTokenHash".

Reply:
updated to use `RevokedTokenHash`